.linelint.yml

```diff
@@ -0,0 +1,14 @@
+# .linelint.yml
+include:
+  - "**/*.js"
+  - "**/*.jsx"
+  - "**/*.ts"
+  - "**/*.tsx"
+exclude:
+  - node_modules/
+  - build/
+rules:
+  eof-newline:
+    enabled: true
+  trim-trailing-whitespace:
+    enabled: true
```

Comment:
I investigated a little bit on the lint fail you have. `no valid rule enabled` You might already know it, but just in case here is what I would try:

- In the workflow containing the lint test for Node, I would have the linelint job as stated in their [README.md](https://github.com/fernandrone/linelint)

```yaml
# .github/workflows/main.yml
on: [push]
name: lint

jobs:
  linelint:
    runs-on: ubuntu-latest
    name: Check if all files end in newline
    steps:
      - name: Checkout
        uses: actions/checkout@v2
      - name: Linelint
        uses: actions/linelint@master
        id: linelint
```

- And then if we look at the end of the file rule, their example looks like this:

```yaml
rules:
  # checks if file ends in a newline character
  end-of-file:
    # set to true to enable this rule
    enable: true

    # set to true to disable autofix (if enabled globally)
    disable-autofix: false

    # will be ignored only by this rule
    ignore:
      - README.md

    # if true also checks if file ends in a single newline character
    single-new-line: true
```
Hope it helps

Reply:
@rngadam do we still need this file ? the[ following action ](https://github.com/ai-cfia/nachet-frontend/blob/c19a1c236f39894ca9b0be2ffe1856fe00fcc7cb/.github/workflows/react-frontend-workflows.yml#L20C1-L22C21 )that is already in the pipeline of the repo is applying linelint : 
```yaml
  repo-standard:
    uses: ai-cfia/github-workflows/.github/workflows/workflow-repo-standards-validation.yml@main
    secrets: inherit
``` 
It was previously not working but I fixed since last week and it should lint the same rules. I dont know if this file was added due to the workflow not working properly so correct me if im wrong.